build.sbt

```diff
@@ -0,0 +1,55 @@
+import Dependencies.appDependencies
+import TestPhases.oneForkedJvmPerTest2
+import Resolvers._
+import uk.gov.hmrc.DefaultBuildSettings.{ addTestReportOption, defaultSettings, scalaSettings }
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
+
```

Comment:
do we need scoverageSettigs for our coverage to take effect?
e.g.
lazy val scoverageSettings = {
  import scoverage.ScoverageKeys
  Seq(
    // Semicolon-separated list of regexs matching classes to exclude
    ScoverageKeys.coverageExcludedPackages := """uk\.gov\.hmrc\.BuildInfo;.*\.Routes;.*\.RoutesPrefix;.*Filters?;MicroserviceAuditConnector;Module;GraphiteStartUp;.*\.Reverse[^.]*""",
    ScoverageKeys.coverageMinimum := 80.00,
    ScoverageKeys.coverageFailOnMinimum := true,
    ScoverageKeys.coverageHighlighting := true,
    parallelExecution in Test := false
  )
}